src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1163,7 +1163,7 @@ table.logs .log-metadata {
 table.logs .log-metadata span {
```

Comment:
this might need to use `> span` too, to avoid affecting spans in the nested dump

Reply:
Yes indeed, I didn't see any changing behavior before so I left it. But it causes some weird wrapping
![image](https://user-images.githubusercontent.com/310006/151545321-79e2296a-df82-4c48-a7c8-354c27cbf7bd.png)
the `$exists...` should start on the same line as the `>`. 

tldr I've added it.